### PR TITLE
Feat/documents

### PR DESCRIPTION
**Pull request by CallumEddisford**

### CODE REVIEW

File: src/Services/Auth.ts [hunk 1]
```diff
@@ -185,26 +200,10 @@ class Auth extends Service
...
-        return this._user;
-    }
-
-    /**
-     * Gets the current business context
-     * @returns Business|void
-     */
-    get business(): Business | null | void {
-        return this._business;
-    }
```
Comment (Smudge3806) [inline src/Services/Auth.ts:207]: Why have these been removed?

**Review by Smudge3806 — COMMENTED** [review]
Great apart from the a question around the deletions in AuthService.

File: src/Services/Auth.ts (hunk 1 above)
Comment (CallumEddisford) [reply src/Services/Auth.ts:207]: Its not

**Comment by CallumEddisford** [comment]
Is this cool to go down @Smudge3806? - I refactored some of the files so getters and stuff are at the top of the documents and functions are underneath, I havent removed any code though

**Review by Smudge3806 — APPROVED** [review]
@CallumEddisford Good to go

**Pull request merged**